src/messaging/messaging-api.ts

```diff
@@ -940,7 +940,11 @@ export interface MessagingOptions {
   [key: string]: any | undefined;
 }
 
-/* Individual status response payload from single devices */
+/**
+ * Individual status response payload from single devices
+ *
+ * @deprecated Returned by {@link Messaging#sendToDevice} which is also deprecated.
```

Comment:
Nit, but "which" is generally preceded by a comma.  Suggest one here and for similar/identical instances below.

Reply:
Good catch! Thanks! Done.